Controllers/param.js

```diff
@@ -0,0 +1,51 @@
+const Playlist = require('../Models/playlist');
+const Video = require('../Models/video');
+const User = require('../Models/user');
+
+// video
+
+exports.getUserById = async (req, res, next, id) => {
+  try {
+    const user = await User.findById(id);
+    if (!user) {
+      throw Error('No such user found');
+    }
+    // console.log({ user });
+    req.user = user;
+    next();
+  } catch (error) {
+    return res.status(400).json({ success: true, error: error.message });
```

Comment:
why is success status true if statusCode is 400?

Reply:
done